src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -672,6 +682,21 @@ class Simplex_tree {
     return simplices_number;
   }
 
+  /**
+   * @brief Returns the dimension of the given sibling simplices.
+   * 
+   * @param curr_sib Pointer to the sibling container.
+   * @return Height of the siblings in the tree (root counts as zero to make the height correspond to the dimension).
+   */
+  int dimension(Siblings* curr_sib) {
+    int dim = 0;
+    while (curr_sib != nullptr) {
+      ++dim;
+      curr_sib = curr_sib->oncles();
+    }
+    return dim - 1;
```

Comment:
Sorry, I was fooled by github interface and created #972 . This can be fixed by: 
```suggestion
    int dim = -1;
    while (curr_sib != nullptr) {
      ++dim;
      curr_sib = curr_sib->oncles();
    }
    return dim;
```